jenkins/PerfCI_Chaos/04_PerfCI_Chaos_Upgrade_OpenShift_Deployment/Jenkinsfile

```diff
@@ -151,6 +151,7 @@ END
                                                         sudo podman run --rm \
                                                             -e UPGRADE_OCP_VERSION="${UPGRADE_OCP_VERSION}" \
                                                             -e UPGRADE_STEP="${upgrade_step}" \
+                                                            -e RUN_TYPE="${RUN_TYPE} \
```

Comment:
Eli, this needs to be fixed.  If it passes, then there's a bug somewhere else, because this is syntactically wrong (there needs to be a closing quote).

Also, UPGRADE_CHANNEL needs double quotes also, not single quotes.